network_modeling/configs/templates/1.2/aruba/common/route-map.j2

```diff
@@ -13,6 +13,11 @@ route-map {{ name }} permit seq {{ sequence.value }}
      {%- for CMN_name, CMN_ip in variables.CMN_IPs.items() if name == CMN_name %}
      set ip next-hop {{ CMN_ip }}
      {%- endfor %}
+{% set sequence.value = sequence.value + 10 %}route-map {{ name }} permit seq {{ sequence.value }}
```

Comment:
This looks like it is being added to the route-map that is used in the default VRF.   This needs to be a different route-map that is applied on the Customer VRF doesn't it?   Because the CAN forwards to the Customer VRF.

Reply:
Has this generated config been tested on one of the CAN systems?

Reply:
Johren-hpe, code changed to effect vrf customer. I did not see your request in time, so I had to re-work the templates.